src/UsgsAstroLsSensorModel.cpp

```diff
@@ -1774,11 +1774,64 @@ void UsgsAstroLsSensorModel::calculateAttitudeCorrection(const double& time, con
     (getValue(7, adj) + getValue(10, adj)* nTime + getValue(13, adj)* nTime2) / m_flyingHeight;
   euler[2] =
     (getValue(8, adj) + getValue(11, adj)* nTime + getValue(14, adj)* nTime2) / m_halfSwath;
-  
+
   calculateRotationMatrixFromEuler(euler, attCorr);
 }
 
 
+// This method works by iteratively adding distortion until the new distorted
+// point, r, undistorts to within a tolerance of the original point, rp.
+void UsgsAstroLsSensorModel::reconstructSensorDistortion(
+    double focalX,

```

Comment:
Don't `focalX` and `focalY` need to be passed by reference so that they will be updated by this function? (Also, if the existing tests didn't catch this, I'm guessing they don't test the distortion?)

Reply:
Yes they do! That is what I meant to do! Good catch!

Reply:
:+1: